CHANGELOG.md

```diff
@@ -1,120 +1,174 @@
+- remove code related to EOL Debian/Ubuntu releases [\#183](https://github.com/voxpupuli/puppet-powerdns/pull/183) ([saz](https://github.com/saz))
+- Remove eol oraclelinux [\#182](https://github.com/voxpupuli/puppet-powerdns/pull/182) ([saz](https://github.com/saz))
+- Remove eol centos [\#180](https://github.com/voxpupuli/puppet-powerdns/pull/180) ([saz](https://github.com/saz))
+- Remove eol redhat [\#179](https://github.com/voxpupuli/puppet-powerdns/pull/179) ([saz](https://github.com/saz))
+- drop EOL Scientific as supported OS [\#177](https://github.com/voxpupuli/puppet-powerdns/pull/177) ([saz](https://github.com/saz))
+
+**Implemented enhancements:**
+
+- Allow puppetlabs/apt 10.x [\#205](https://github.com/voxpupuli/puppet-powerdns/pull/205) ([smortex](https://github.com/smortex))
+- metadata.json: Add OpenVox [\#200](https://github.com/voxpupuli/puppet-powerdns/pull/200) ([jstraw](https://github.com/jstraw))
+- allow puppetlabs/mysql 16.X [\#197](https://github.com/voxpupuli/puppet-powerdns/pull/197) ([saz](https://github.com/saz))
+- update recursor version to 5.0, authoritative version to 4.9 [\#195](https://github.com/voxpupuli/puppet-powerdns/pull/195) ([saz](https://github.com/saz))
+- replace params.pp with hiera data [\#184](https://github.com/voxpupuli/puppet-powerdns/pull/184) ([saz](https://github.com/saz))
+- add RedHat/CentOS/OracleLinux 8/9 and Debian 11/12 as supported OS [\#178](https://github.com/voxpupuli/puppet-powerdns/pull/178) ([saz](https://github.com/saz))
+
+**Fixed bugs:**
+
+- run sqlite backend table creation as user pdns, fixes \#211 [\#212](https://github.com/voxpupuli/puppet-powerdns/pull/212) ([saz](https://github.com/saz))
+- replace hard-coded pdns group, fixes \#202 [\#209](https://github.com/voxpupuli/puppet-powerdns/pull/209) ([saz](https://github.com/saz))
+- fix: parameter type error messages [\#201](https://github.com/voxpupuli/puppet-powerdns/pull/201) ([pdemonaco](https://github.com/pdemonaco))
+- require package before trying to manage config file [\#198](https://github.com/voxpupuli/puppet-powerdns/pull/198) ([saz](https://github.com/saz))
+- set "undef" as default value for backend packages [\#196](https://github.com/voxpupuli/puppet-powerdns/pull/196) ([saz](https://github.com/saz))
+- fix schema file paths on RedHat based OS [\#194](https://github.com/voxpupuli/puppet-powerdns/pull/194) ([saz](https://github.com/saz))
+- remove unused powertools yum repo [\#191](https://github.com/voxpupuli/puppet-powerdns/pull/191) ([saz](https://github.com/saz))
+- correct pdns.conf permissions on Redhat derivatives [\#185](https://github.com/voxpupuli/puppet-powerdns/pull/185) ([pdemonaco](https://github.com/pdemonaco))
+- pin recursor apt source with higher priority [\#176](https://github.com/voxpupuli/puppet-powerdns/pull/176) ([saz](https://github.com/saz))
+
+**Closed issues:**
+
+- sqlite3 backend exec should be run as user `pdns` [\#211](https://github.com/voxpupuli/puppet-powerdns/issues/211)
```

Comment:
addes skip-changelog here because we have the related PR already in the CHANGELOG.md